web/client/plugins/Map.jsx

```diff
@@ -27,6 +27,7 @@ import mapTypeReducer from "../reducers/maptype";
 import additionalLayersReducer from "../reducers/additionallayers";
 import mapEpics from "../epics/map";
 import pluginsCreator from "./map/index";
+import createFeatureFilter from '../utils/FilterUtils';
```

Comment:
This causes an error when trying to load a vector layer.
```suggestion
import {createFeatureFilter} from '../utils/FilterUtils';
```